.circleci/config.yml

```diff
@@ -231,9 +239,21 @@ jobs:
           force_native_shadow_mode: true
       - run_karma:
           enable_native_custom_element_lifecycle: true
+      - run_karma:
+          enable_scoped_custom_element_registry: true
+      - run_karma:
+          disable_synthetic: true
+          enable_native_custom_element_lifecycle: true
+      - run_karma:
+          disable_synthetic: true
+          enable_scoped_custom_element_registry: true
+      - run_karma:
+          enable_native_custom_element_lifecycle: true
+          enable_scoped_custom_element_registry: true
       - run_karma:
           disable_synthetic: true
           enable_native_custom_element_lifecycle: true
+          enable_scoped_custom_element_registry: true
```

Comment:
I'm not a huge fan of this situation, but I don't have a good solution. The problem with having a lot of flags is that you need to test every combo to get good coverage.